tests/bsim/bluetooth/host/misc/sample_test/test_scripts/run.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
+
+# Provides common functions for bsim tests.
+# Mainly `Execute`, and `wait_for_background_jobs`.
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+# Helper variable. Expands to "tests_bsim_bluetooth_host_misc_sample_test".
+test_name="$(guess_test_long_name)"
+
+# Use a unique simulation id per test script. It is a necessity as the CI runner
+# will run all the test scripts in parallel. If multiple simulations share the
+# same ID, they will step on each other's toes in unpredictable ways.
+simulation_id=${test_name}
+
+# This controls the verbosity of the simulator. It goes up to 9 (not 11, sorry)
+# and is useful for figuring out what is happening on the PHYsical layer (e.g.
+# device 1 starts listening at T=100us) among other things.
+# `2` is a good default.
+verbosity_level=2
+
+# This sets the watchdog timeout for the `Execute` function.
+#
+# Although all simulations are started with a bounded end (the `sim_length`
+# option), something very wrong can still happen and this additional time-out
+# will ensure all executables started by the current script are killed.
+#
+# It measures wall-clock time, not simulated time. E.g. a test that simulates 5
+# hours might actually complete (have a runtime of) 10 seconds.
+#
+# The default is set in `sh_common.source`.
+# Guidelines to set this value:
+# - Do not set it to a value lower or equal to the default.
+# - If the test takes over 5 seconds of runtime, set `EXECUTE_TIMEOUT` to at
+#   least 5 times the run-time on your machine.
+EXECUTE_TIMEOUT=120
+
+# Set simulation length, in microseconds. The PHY will run for this amount of
+# simulated time, unless both devices exit.
+#
+# If you are not early-exiting the devices (e.g. using `TEST_PASS_AND_EXIT()`),
+# please run the test once and set the simulation time in the same ballpark. No
+# need to simulate hours of runtime if the test finishes in 10 seconds.
+#
+# This supports up to (2^32) - 1 us, roughly 70 minutes.
+# In our case, the test completes in less than 2s of simulated time.
+SIM_LEN_US=$((2 * 1000 * 1000))
```

Comment:
Is this extra variable really useful?

Reply:
I think so. You'd probably want to always use the same simulation length for each `test_exe`, and in that case I think it's always better to use a single variable rather than 2 literal values that will always be the same value

Reply:
Now it became very irrelevant :)